tests/unit/test_s3.py

```diff
@@ -0,0 +1,204 @@
+from unittest.mock import Mock, patch
+
+import pandas as pd
+
+from sdgym.s3 import is_s3_path, parse_s3_path, write_csv, write_file
+
+
+def test_is_s3_path_with_local_dir():
+    """Test the ``sdgym.s3.is_s3_path`` function with a local
```

Comment:
A small style detail: let's try to follow the docstring format recommendations from [PEP 257](https://www.python.org/dev/peps/pep-0257/), which means that the first line is a complete sentence, ending with a dot `'.'`, and any additional information is in a line below separated by a blank line.

Here I would say
```
"""Test the ``sdgym.s3.is_s3_path`` function with a local path.

If the given path is not an S3 path, it should return ``False``.
```